freqtrade/freqtradebot.py

```diff
@@ -553,7 +552,7 @@ def execute_buy(self, pair: str, stake_amount: float, price: Optional[float] = N
 
         if not strategy_safe_wrapper(self.strategy.confirm_trade_entry, default_retval=True)(
                 pair=pair, order_type=order_type, amount=amount, rate=buy_limit_requested,
-                time_in_force=time_in_force):
+                time_in_force=time_in_force, current_time=datetime.utcnow()):
```

Comment:
you will want to use `datetime.now(timezone.utc)` here - otherwise comparisons will fail with "cannot compare timezone aware and timezone native ..."